### PR TITLE
fix(curriculum): verbiage

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

@miyaliu666 caught this on crowdin that two tests in this challenge, which test different things, used the same wording. I've adjusted the wording to clarify the assertions each test runs.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md [hunk 1]
```diff
@@ -59,13 +59,13 @@ Add a new album to the `myMusic` array. Add `artist` and `title` strings, `relea
 assert(Array.isArray(myMusic));
 ```
 
-`myMusic` should have at least two objects
+`myMusic` should have at least two values
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md:62]:
```suggestion
`myMusic` should have at least two elements
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md [hunk 2]
```diff
@@ -59,13 +59,13 @@ Add a new album to the `myMusic` array. Add `artist` and `title` strings, `relea
...
 
 ```js
 assert(myMusic.length > 1);
 ```
 
-`myMusic` should contain at least two objects
+The values in the `myMusic` array should be objects
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md:68]:
```suggestion
The elements in the `myMusic` array should be objects
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thanks, for this. I should have noticed in the last PR.

The term `values` does not sit well for arrays, IMO.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md (hunk 1 above)
Comment (ojeytonwilliams) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md:62]: I feel like entries, items and values are all more or less fine.  I had a quick glance through the curriculum and we use all three.

Comment (ShaunSHamilton) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md:62]: Right, just looking at [Mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array), they do not use `values` at all to refer to the elements in the array.

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md:62]: Elements works for me 🙂

**Review by ShaunSHamilton — APPROVED** [review]
Thanks, for this, Nich.
LGTM 👍

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md (hunk 1 above)
Comment (ojeytonwilliams) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md]: Yeah, sorry. Wasn't really worth my bringing up.

Elements are fine.

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**